mod_reforged/hooks/config/strings.nut

```diff
@@ -1665,9 +1672,10 @@ foreach (vanillaDesc in vanillaDescriptions)
 			Description = [
 				"Damage to [Hitpoints|Concept.Hitpoints] is reduced by " + ::MSU.Text.colorNegative("30%") + " and to Armor by " + ::MSU.Text.colorNegative("20%") + ".",
 				"The bonus drops exponentially when wearing head and body armor with a total penalty to [Maximum Fatigue|Concept.MaximumFatigue] above 35.",
-				"When attacking a target against whom you have a [Reach Disadvantage,|Concept.ReachAdvantage] reduce this disadvantage by " + ::MSU.Text.colorPositive(1) + " if your [Initiative|Concept.Initiative] is higher than that of your target."
+				"Does not affect damage from mental attacks or status effects, but can help to avoid receiving them.",
+				"When attacking a target against whom you have a [Reach Disadvantage,|Concept.ReachAdvantage] reduce this disadvantage by " + ::MSU.Text.colorPositive(1) + " if your [Initiative|Concept.Initiative] is greater than that of your target."
 				"[Brawny|Perk+perk_brawny] does not affect this perk.",
-				"Cannot be picked if you have [Nimble.|Perk+perk_nimble]"
+				"Cannot be taken if you have [Nimble.|Perk+perk_nimble]"
```

Comment:
Same comment about taken vs picked as the other places. Just marking it, no need for comment on this conversation.